content/datenschutz.md

```diff
@@ -119,7 +119,7 @@ Datenschutzerklärung
 
 <h3>VIII. Internetspezifische Datenverarbeitungen</h3>
 
-<p>Während der Nutzung unserer Internetseiten werden durch uns automatisch technische Zugriffsdaten aufgezeichnet und ausgewertet. Hierfür benutzen wir das Tool "Matomo" (ehemals „Piwik“), ein Open-Source-Programm für Webanalytik, welches statistische Kennwerte zur Nutzung unserer Website auswertet. Diese Daten können jedoch nicht einer bestimmten Person zugeordnet werden; der einzelne Benutzer bleibt hierbei anonym. Weitere Informationen zu "Matomo" finden Sie auf der Website
+<p>1. "Matomo": Während der Nutzung unserer Internetseiten werden durch uns automatisch technische Zugriffsdaten aufgezeichnet und ausgewertet. Hierfür benutzen wir das Tool "Matomo" (ehemals „Piwik“), ein Open-Source-Programm für Webanalytik, welches statistische Kennwerte zur Nutzung unserer Website auswertet. Diese Daten können jedoch nicht einer bestimmten Person zugeordnet werden; der einzelne Benutzer bleibt hierbei anonym. Weitere Informationen zu "Matomo" finden Sie auf der Website
```

Comment:
So wie es aussieht, ist aktuell kein Matomo in die Seite eingebunden.

Reply:
Danke für den Hinweis. Auf die Frage, wie wir damit umgehen sollen, gab es ja noch keine eindeutige Antwort im Slack. Dann lassen wir es hier erst einmal abgeschaltet und ich nehme es in der Erklärung raus.